lib/sober_swag/output_object/field_syntax.rb

```diff
@@ -37,10 +37,11 @@ def primitive(name)
       # Note that merging in a full output object *will not* also merge in views, just fields defined on the base.
       #
       # @param other [#fields] a field container, like a {SoberSwag::OutputObject} or something
+      # @param opts [Hash] accepts a key of :except to optionally exclude a field from the output object being merged
```

Comment:
This should probably be documented in the `docs/serializer.md` file (I think that's where the merge method is currently documented?)